### PR TITLE
Pharmgkb schema

### PR DESCRIPTION
**Pull request by clincoln8**

### CODE REVIEW

File: schema/PharmGKBIDs.mcf
Comment (tjann) [inline schema/PharmGKBIDs.mcf:2]: Can you expand the note a bit more to say how modified by hand?

File: schema/PharmGKBIDs.mcf
Comment (tjann) [inline schema/PharmGKBIDs.mcf:4]: I'll leave it to @spiekos -- typically it's nice to have descriptions for properties so that people browsing the schema know when to reuse. In the case of Bio, I'm not sure how "obvious" alfred, genAtlas, etc. are. But perhaps at least a hint that these are IDs from various bio databases?

File: schema/CompoundSchemaEnum.mcf
Comment (tjann) [inline schema/CompoundSchemaEnum.mcf:2507]: I wonder if it's better to name this as RelationshipEvaluationResult? It's interesting to say that a relationship is type of "not associated", since "not associated" means "no relationship"?

File: schema/CompoundSchemaEnum.mcf
Comment (tjann) [inline schema/CompoundSchemaEnum.mcf:2542]: should be descriptionUrl-- also below; could you check to make sure that we have descriptionUrl not url for the rest of this PR?

**Review by tjann — COMMENTED** [review]
Nice, this was very clean. Thanks! I'll approve when comments are resolved.

File: schema/CompoundSchema.mcf
Comment (spiekos) [reply schema/CompoundSchema.mcf:32]: I'm conflicted on this. dcs:GeneticAssoication could work. However, it's a subClass of BiologicalElement and thus far we've been treating BiologicalElement and ChemicalCompound as two separate trees. Here they are combining the two. I'm more inclined to not put it under either tree. What do you think @tjann

File: schema/CompoundSchemaEnum.mcf
Comment (spiekos) [reply schema/CompoundSchemaEnum.mcf:2507]: I think either works

File: schema/GenomeAnnotation.mcf
Comment (spiekos) [reply schema/GenomeAnnotation.mcf:562]: The bio schema mainly follows the no space convention. I'm ok with enforcing this as standard, unless you want to switch to space convention @tjann

File: schema/PharmGKB.mcf
Comment (spiekos) [reply schema/PharmGKB.mcf:17]: Whatever you decide to name the RelationshipAssociationEnum or the RelationshipEvaluationResultEnum keep this property name consistent here.

File: schema/PharmGKBIDs.mcf
Comment (spiekos) [reply schema/PharmGKBIDs.mcf:2]: +1

File: schema/PharmGKBIDs.mcf
Comment (spiekos) [reply schema/PharmGKBIDs.mcf:4]: These were generated by a script. It would require manual work to go and extract URLs and descriptions for each of these script generated properties. I'm ok not having them to avoid doing this.

File: schema/PharmGKBIDs.mcf
Comment (spiekos) [inline schema/PharmGKBIDs.mcf:68]: @tjann, since a Drug is a subClass of ChemicalCompound should we just list the domain as ChemicalCompound or should we list both?

**Review by spiekos — CHANGES_REQUESTED** [review]

File: schema/GenomeAnnotation.mcf
Comment (tjann) [reply schema/GenomeAnnotation.mcf:562]: SG. Let's do no space then. I'm also not attached to this.

File: schema/PharmGKBIDs.mcf
Comment (tjann) [reply schema/PharmGKBIDs.mcf:68]: Yeah, we can just list ChemicalCompound

File: schema/CompoundSchema.mcf
Comment (tjann) [reply schema/CompoundSchema.mcf:32]: Ah I see. I saw Gene and thought BiologicalElement would be good.

Maybe it's still good to create an umbrella class. Say, ChemicalElement, BioChemicalElement.. or if not Element, Concept/Thing?

Comment (spiekos) [reply schema/CompoundSchema.mcf:32]: I think we should just make it either a Thing or make a BiomedicalElement (subclass of Thing) that BiologicalElement and ChemicalElement. I'm liking the second option, because I could also see making a MedicalElement class in the future as well, which would fall under the BiomedicalElement umbrella.

Comment (spiekos) [reply schema/CompoundSchema.mcf:32]: Let's make a new class BiomedicalElement as a subclass of Thing and then add this and BiologicalElement and ChemicalElement as a subclass of BiomedicalElement.

**Review by spiekos — CHANGES_REQUESTED** [review]

**Comment by clincoln8** [comment]
CompoundSchema.mcf: added BiomedicalElement node to the top of the file, make ChemicalCompound and ChemicalCompoundGenomeAssociation both subClasses of BiomedicalElement

GenomeAnnotation.mcf: made BiologicalElement a subClass of BiomedicalElement

File: schema/PharmGKBIDs.mcf
Comment (tjann) [reply schema/PharmGKBIDs.mcf]: Thanks!

File: schema/CompoundSchema.mcf
Comment (tjann) [inline schema/CompoundSchema.mcf:12]: I see this is not from you, but could you also change all url occurrences in this file to descriptionUrl?

**Review by tjann — APPROVED** [review]
LGTM, but will let Sam give a final stamp approval before getting this in.

File: schema/CompoundSchema.mcf
Comment (clincoln8) [reply schema/CompoundSchema.mcf]: done!

Comment (tjann) [reply schema/CompoundSchema.mcf]: Thanks!

**Pull request merged**
